src/MCPClient/tests/test_create_dataverse_mets.py

```diff
@@ -211,3 +251,31 @@ def test_mets_content(self, fixture, tmpdir):
                 pytest.fail("Unexpected MDREF found in metadata: {}"
                             .format(ref.get("LABEL")))
         assert dataset_json is True
+
+    @pytest.mark.parametrize(
+        "fixture", [dv_1, dv_2, dv_3, dv_4, dv_5, dv_6, dv_7, dv_8])
+    def test_ddi_authors(self, fixture, tmpdir):
+        """Test that we have retrieved the correct number of authors and that
+        the list compares 1:1 in order.
+        """
+        mets_path = self._create_mets(fixture.fname, tmpdir)
+        if not os.path.isfile(mets_path):
+            pytest.fail("Fixtures were not previously setup correctly.")
+
+        try:
+            mets = metsrw.METSDocument.fromfile(mets_path)
+        except mets.MetsError:
+            pytest.fail("Could not parse mets {}".format(mets_path))
+
+        mets_root = mets.serialize()
+
+        # Setup namespaces to search for in the document.
```

Comment:
This is a grammar question from a ESL out of curiosity, not really suggesting a change.
Shouldn't this read `Set up` (the verb) instead of `Setup`?

Reply:
For a native speaker, my English is pretty bad @sevein. It's more fluid than perhaps it should be for something with pretty clear rules and structures! (probably like some the code you see from me! :innocent: ) So this is a great question, and, today I discovered a Stack Overflow for the english language! https://english.stackexchange.com/a/3665 - and you are correct! :nerd_face: